bucket/adguardhome.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "0.107.65",
+    "description": "Network-wide ads & trackers blocking DNS server",
+    "homepage": "https://github.com/AdguardTeam/AdGuardHome",
```

Comment:
Adguard Home has its own homepage in https://adguard.com/en/adguard-home/overview.html